include/global.h

```diff
@@ -101,6 +101,12 @@
 // with for the calculation
 #define GBA_FRAMES_PER_SECOND 60
 
+#define SWAP_AND_NEGATE(a, b)                                                           \
```

Comment:
@jiangzhengwenjz have you seen this anywhere in katam? Is there some one liner we can use here instead?

Reply:
> @jiangzhengwenjz have you seen this anywhere in katam? Is there some one liner we can use here instead?

In katam it's juts swap. No negate.

Reply:
So what @JaceCear has written here can’t be simplified? I wondered if there was some weird trick in agbcc

Reply:
> So what @JaceCear has written here can’t be simplified? I wondered if there was some weird trick in agbcc

https://github.com/jiangzhengwenjz/katam/blob/master/include/global.h#L53 This is what I have in katam. So it's really different.